src/bin/hamcore/openvpn_sample.ovpn

```diff
@@ -102,8 +102,8 @@ $TAG_BEFORE_REMOTE$remote $TAG_HOSTNAME$ $TAG_PORT$
 # It is recommended to use a better algorithm if your client supports it.
 # When you specify an unsupported algorithm, an error will occur.
 
-cipher AES-128-CBC
-auth SHA1
+;cipher AES-128-CBC
```

Comment:
OpenVPN 2.4 still uses `BF-CBC` as default cipher, I don't think it's a good idea to comment out the setting.

Reply:
OK, I will make more investigation